test_team_3.py

```diff
@@ -47,6 +47,15 @@ def test_us_bank_number(self):
 
     def test_es_nif(self):
         """Test es_nif functionality"""
+        #Negative Testcase Should not replace id
+        test_string = "documento nacional de identidad B112345267A8A"
```

Comment:
If possible can you provide comments for the code since it is in spanish.

Reply:
[Here](https://github.com/microsoft/presidio/blob/db8ff8254123a113a0d511a484647734021de612/presidio-analyzer/presidio_analyzer/predefined_recognizers/es_nif_recognizer.py#L19) is the regex used to detect a valid identifier